versions/unversioned/guides/using-apollo.rst

```diff
@@ -0,0 +1,182 @@
+.. _using-apollo:
+
+Using Apollo
+============
+
+The `Apollo <http://www.apollodata.com/>`_ client enables you to use
+`GraphQL <http://graphql.org/>`_ in your Exponent applications. Unlike other
+GraphQL clients, Apollo works out of the box with Exponent.
+
+Why GraphQL?
+^^^^^^^^^^^^
+
+GraphQL is a query language for your API designed and released by Facebook. The
+reason why GraphQL is interesting to so many developers is that it improves upon
+critiques of other API architectures like SOAP and REST.
+
+For instance, with a GraphQL query you are guaranteed to only ever get the
+*exact data you need*. No more overfetching or underfetching from API endpoints.
+Not only does a GraphQL query allow you to get exactly the data you need, but it
+also allows you to do so in one request.
+
+To learn more about GraphQL see `graphql.org <http://graphql.org/>`_.
+
+Why Apollo?
+^^^^^^^^^^^
+
+Apollo is not the only library designed to help you manage your API data.
+However, the principles which Apollo was designed around make it a powerful yet
+still flexible choice. Some of those principles are for Apollo to be:
+
+1. Incrementally adoptable.
```

Comment:
These are a bit vague - I think we should make them more detailed or change them to something Exponent users would care more about.